### ISSUE TITLE
Quick filter params are not interceptable via transformParams()

### ISSUE THREAD

**Issue by endel** [marmelab/ng-admin#188]
When I'm using quick filters it isn't possible to intercept it on `transformParams` method. 

**Example:**

```
listView.addQuickFilter('Published', function () {
    return {
        published: true
    };
});
```

```
app.transformParams(function(params) {
    // only 'params.page' and 'params.per_page' are available
    return params; 
})
```

After selecting the `Published` quick filter, only `page` and `per_page` are available on `transformParams`.

Shouldn't `params.published` be available too?

**Comment by fzaninotto** [comment]
Fixed by #211

**Comment by endel** [comment]
Awesome @fzaninotto!
